docs/en/04_Changelogs/4.11.0.md

```diff
@@ -23,6 +24,33 @@ In accordance with our [PHP support policy](/Getting_Started/Server_Requirements
 
 ## Features and enhancements {#features-and-enhancements}
 
+### Upload and use WebP images {#webp}
+
+WebP is an alternative image format for displaying picture on websites. It provides generally better results in most use cases to JPEG and PNG. 
+
+Read [An image format for the Web](https://developers.google.com/speed/webp) for more information about WebP.
+
+WebP has wide – but not universal – support across web browsers. Internet Explorer is the main browser that does not support WebP at this stage.
+
+Read [Can I use WebP?](https://caniuse.com/webp) to see which browsers can render WebP images.
+
+Until now, Silverstripe CMS would default to blocking content authors from uploading WebP images. Given that Internet Explorer is soon to be end-of-life and that its market share are now under 1% according to most surveys, we decided the time had come to enable WebP by default in the CMS.
```

Comment:
Maybe worth specifically mentioning that Internet Explorer goes EOL in June?
https://blogs.windows.com/windowsexperience/2021/05/19/the-future-of-internet-explorer-on-windows-10-is-in-microsoft-edge/

Reply:
```suggestion
Until now, Silverstripe CMS would default to blocking content authors from uploading WebP images. Given that [Internet Explorer will be end-of-life in June 2022](https://blogs.windows.com/windowsexperience/2021/05/19/the-future-of-internet-explorer-on-windows-10-is-in-microsoft-edge/) and that its market share are now under 1% according to most surveys, we decided the time had come to enable WebP by default in the CMS.
```